src/integration/pages/articles/crossPlatformTests.js

```diff
@@ -3,18 +3,21 @@ import { runCommonCrossPlatformTests } from '../../common';
 export default () => {
   runCommonCrossPlatformTests();
 
-  it('I can see an image with a caption', () => {
-    const imageEl = document.querySelector(
-      'main figure img, main figure amp-img',
-    );
-    const imageCaptionEl = document.querySelector('main figure figcaption');
+  // These service's urls don't have images with captions. Need to find better examples
+  if (!['zhongwen', 'uzbek', 'serbian', 'ukchina'].includes(service)) {
```

Comment:
Another alternative would be to move the imageEl code outside the test, and only run this test if an image is found?

I don't believe that articles need to have an image, but I could be mistaken.